doc/5.reference/soundfiler-help.pd

```diff
@@ -122,56 +141,37 @@
-#X obj 863 393 cnv 19 323 118 empty empty empty 20 12 0 12 #e0e0e0 #404040 0;
-#N canvas 0 22 450 300 (subpatch) 0;
-#X array array1 78003 float 2;
-#X coords 0 1 78003 -1 325 120 1 0 0;
-#X restore 862 248 graph;
-#N canvas 0 22 450 300 (subpatch) 0;
-#X array array2 78003 float 2;
-#X coords 0 1 78003 -1 325 120 1 0 0;
-#X restore 862 392 graph;
-#N canvas 460 493 480 224 8-byte 0;
-#X text 19 62 The precision of the 8-byte samples is based on the precision of the Pure Data build. If Pd is single precision (using 32-bit float internally) \, the written samples will be cast from 4-byte float to 8-byte double when writing to a file with -bytes 8 argument. In this case it's better to write with 4-byte float samples for a smaller file with the same precision., f 63;
-#X text 19 159 if Pd is double-precision \, writing 8-byte float samples will be full precision as Pd uses 64-bit float internally., f 63;
-#X text 19 19 Writing 8-byte (ie. 64-bit) floating point samples are supported for the .wave \, .aiff/.aifc and .caf file types.;
-#X restore 440 503 pd 8-byte float;
-#X text 18 540 -normalize (normalize file to '1' \, note that arrays whose values exceed the -1 to 1 range are automatically normalized), f 71;
-#X text 611 547 # samples;
-#X text 696 548 info: sample rate \, header size \, channels \, bytes \, endianness;
-#X text 27 83 -maxsize <float> (maximum number of samples to resize to *), f 73;
-#X text 27 66 -resize (resizes arrays to the size of the sound file *), f 73;
-#X text 1043 131 read overriding the file's header, f 17;
-#X text 17 288 * When loading a file \, [soundfiler] populates the array until reaching its size. If the sound file is smaller \, the remaining points are zeroed out. The -resize flag resizes the array to the sample size \, while the -maxsize flag sets a maximum size to clip to even if used in conjunction with the '-resize' flag (also \, errors are given to warn you about the truncation). The -maxsize was needed for historical reasons when its default value used to be rather small (4 million). The default now is much bigger \, hence \, this flag can just be ignored., f 78;
-#X text 18 485 -nframes <float> (maximum points to write \, default from skip point to end of array), f 83;
-#X text 18 503 -bytes <float> (bytes per sample: 2 \, 3 \, 4 \, or 8 default: 2);
-#X connect 0 0 11 0;
-#X connect 0 1 8 0;
-#X connect 9 0 0 0;
-#X connect 10 0 0 0;
-#X connect 12 0 23 0;
+#X restore 663 384 pd more-write-examples;
+#X text 27 204 -ascii (read a file containing ascii numbers), f 73;
+#X text 48 221 This may only be combined with '-resize'. Spaces or newlines are valid separators. Non-numeric fields get replaced by zero. If multiple arrays are specified \, the first elements of each array should come first in the file \, followed by all the second elements and so on (interleaved). Files must not contain a BOM (when saved with UTF-8 encoding)., f 71;
+#X connect 0 0 9 0;
```

Comment:
i'm not entirely sure whether people who do not know what `-ascii` refers to, will be able to decode the acronym `BOM`.